### DISCUSSION TITLE
Version picker disappears when a new version is added to version.json

### DISCUSSION THREAD

**Discussion by syeda-git** [squidfunk/mkdocs-material#3582] [Q&A] [answered]
I am playing around with the versioning [project](https://github.com/squidfunk/mkdocs-material-example-versioning).

Added a new 0.3 folder with a new file in it. However, when i add 0.3 in the versions.json file, the version picker disappears from the header.
https://github.com/syeda-git/mkdocs-sandbox/tree/version

Can somebody point out the issue? I must be missing something obvious.

Thanks...
Syed

**Comment by squidfunk** [comment]
Hmm, I looked at your site but I don't see anything obvious. The `versions.json` is loaded, but the version picker isn't there, yes. Please update to the latest version of Material for MkDocs and check whether the issue persists. There might have been some fixes and/or changes to the version selector lately, so it might fix the issue at hand.

**Comment by syeda-git** [comment]
Upgraded and rebuilt. But, the issue persists. The Version picker seems to take only two versions for some reason. Note that the 'latest' folder contains 0.3 but the Version picker doesn't show that.
Any ideas?

**Reply by squidfunk** [reply]
Nope, nothing from the top of my head. There are many users using it successfully.

Did you set a [default version](https://github.com/jimporter/mike#setting-the-default-version)? Other than that, maybe @jimporter has an idea.

**Comment by syeda-git** [comment]
The default version is set now. Is it possible for you to update your [project](https://github.com/squidfunk/mkdocs-material-example-versioning) to include a third version?
@jimporter any idea?

**Comment by syeda-git** [comment, marked as answer]
Strange as it may sound. Suddenly, all 3 versions now show up correctly. Maybe there was some minor issue in the json file before. Not sure what happened exactly.

**Reply by squidfunk** [reply]
Okay great. Considering it resolved.
